N/normaliz/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "normaliz"
+version = v"3.8.4"
+
+# Collection of sources required to build normaliz
+sources = [
+    ArchiveSource("https://github.com/Normaliz/Normaliz/releases/download/v$version/normaliz-$version.tar.gz",
+    "80d21ebaf1a2d472ccdc1e1b2e42b4d71f45f3b8df4d7195ff83edf38f8945c8")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd normaliz-*
+# avoid libtool problems
+rm /workspace/destdir/lib/libgmpxx.la
+# workaround for #624: remove too old libstdc++ from CompilerSupportLibraries
+rm -f /workspace/destdir/lib/libstdc++*
+./configure --prefix=$prefix --host=$target --build=${MACHTYPE} --with-gmp=$prefix CPPFLAGS=-I$prefix/include LDFLAGS=-L$prefix/lib
+make -j
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# windows build would require MPIR instead of GMP for 'long long'
```

Comment:
Normaliz explicitly includes MPIR from `libnormaliz/general.h` instead of GMP on Windows but julia uses GMP on all platforms. 

I'm quite sure it is possible to build it on Windows, but it doesn't really fit the ecosystem like this and was not worth the effort for me as most other packages that I need don't support windows anyway.
There is even a BinaryBuilder for MPIR but I don't want to deal with runtime issues when one library passes an MPIR number to a different library that uses GMP.

`libnormaliz/general.h`:
```C
#ifdef _WIN32     // for 32 and 64 bit windows
#define NMZ_MPIR  // always use MPIR
#endif

#ifdef NMZ_MPIR  // use MPIR
#include <mpirxx.h>
#else  // otherwise use GMP
#include <gmpxx.h>
#endif
```

I don't remember the details from when I first added this restriction but removing the `NMZ_MPIR` define now gives plenty of errors like this:
```
./libnormaliz/HilbertSeries.cpp:1122:26: error: ambiguous overload for ‘operator*’ (operand types are ‘size_t {aka long long unsigned int}’ and ‘__gnu_cxx::__alloc_traits<std::allocator<__gmp_expr<__mpz_struct [1], __mpz_struct [1]> > >::value_type {aka __gmp_expr<__mpz_struct [1], __mpz_struct [1]>}’)
./libnormaliz/cone.cpp:5322:18: error: ambiguous overload for ‘operator*=’ (operand types are ‘mpz_class {aka __gmp_expr<__mpz_struct [1], __mpz_struct [1]>}’ and ‘size_t {aka long long unsigned int}’)
./libnormaliz/cone.cpp:5865:26: error: ambiguous overload for ‘operator=’ (operand types are ‘mpq_class {aka __gmp_expr<__mpq_struct [1], __mpq_struct [1]>}’ and ‘size_t {aka long long unsigned int}’)
./libnormaliz/cone.cpp:5865:26: error: ambiguous overload for ‘operator=’ (operand types are ‘mpq_class {aka __gmp_expr<__mpq_struct [1], __mpq_struct [1]>}’ and ‘size_t {aka long long unsigned int}’)
./libnormaliz/cone.cpp:6771:18: error: ambiguous overload for ‘operator=’ (operand types are ‘mpq_class {aka __gmp_expr<__mpq_struct [1], __mpq_struct [1]>}’ and ‘size_t {aka long long unsigned int}’)
./libnormaliz/full_cone.cpp:3050:38: error: conversion from ‘size_t {aka long long unsigned int}’ to ‘const value_type {aka const __gmp_expr<__mpz_struct [1], __mpz_struct [1]>}’ is ambiguous
./libnormaliz/full_cone.cpp:6036:5: error: conversion from ‘size_t {aka long long unsigned int}’ to ‘std::vector<__gmp_expr<__mpz_struct [1], __mpz_struct [1]> >::value_type {aka __gmp_expr<__mpz_struct [1], __mpz_struct [1]>}’ is ambiguous
```